spec/support/aruba.rb

```diff
@@ -18,7 +18,7 @@ def dirs
   c.include SpecHelper::Aruba
```

Comment:
I recommend to replace this file with this:

~~~ruby
require 'aruba/rspec`
~~~

And then add `type: :aruba` to all examples which make use of `aruba`'s methods or use shared examples which make use of aruba, e.g. [this one](https://github.com/jasonlong/geo_pattern/blob/fix-travis/spec/support/shared_examples/pattern.rb#L1).

~~~ruby
Rspec.describe 'My feature', type: :aruba do
end
~~~

Otherwise you  get errors, that a method is missing.